earth_enterprise/src/fusion/autoingest/tools/genewimageryresource.cpp

```diff
@@ -290,7 +290,7 @@ main(int argc, char *argv[]) {
 
       // validate mosaic inputs
       CmdLine cmdline;
-      cmdline << "gevirtualraster" << "--validate" << "-o" << "foo.khvr";
+      cmdline << "/opt/google/bin/gevirtualraster" << "--validate" << "-o" << "foo.khvr";
```

Comment:
Is there a way to do this that wouldn't cause problems if we start installing somewhere other than `/opt/google/bin`? For example, could we just add `/opt/google/bin` to the path when we run the command, or run it from the same directory as the calling binary, or is there some configuration file we can get /opt/google/bin from, or something like that? I can live with it this way but it would be nice to have it a bit more flexible.

Reply:
I decided this way, as there are lot of places in code, which uses this absolutepath.